vllm/v1/engine/async_llm.py

```diff
@@ -144,25 +144,26 @@ async def add_request(
         """Add new request to the AsyncLLM."""
 
         # 1) Create a new output queue for the request.
-        if self.output_processor.is_request_active(request_id):
-            raise ValueError(f"Request id {request_id} already running.")
```

Comment:
Curious of the reason for removing this check?

Reply:
I think I was pondering whether `is_request_active()` should check the parent request ID or the child request ID, and then I noticed in `OutputProcessor`:

```
    def add_request(
        self,
	request: EngineCoreRequest,
        parent_req: ParentRequest,
        request_index: int,
	queue: Optional[asyncio.Queue[RequestOutput]] = None,
    ) -> None:
	request_id = request.request_id
        if request_id in self.request_states:
            raise ValueError(f"Request id {request_id} already running.")
```

So it seems completely redundant?